apps/web-giddh/src/meta/magic.html

```diff
@@ -164,16 +163,27 @@
                                                                                     style="font-size:15px;"></i>
                                                                             </span>
                                                                         </template>
+<!--                                                                        <template-->
```

Comment:
Not removing commented code because it may use later on after sales team approval